apps/docs/src/styles/global.css

```
@@ -1,91 +1,73 @@
+  --shadow-sm: 0 4px 8px rgba(9, 9, 11, 0.08);
+  --shadow-md: 0 8px 24px rgba(9, 9, 11, 0.12);
+  --shadow-lg: 0 24px 60px -28px rgba(9, 9, 11, 0.3);
 
-  --shadow-xs: 0px 2px 4px rgba(0, 0, 0, 0.08);
-  --shadow-sm: 0px 4px 8px rgba(0, 0, 0, 0.125);
-  --shadow-md: 0px 8px 16px rgba(0, 0, 0, 0.125);
-  --shadow-lg: 0px 16px 32px rgba(0, 0, 0, 0.125);
+  --font-display: 'Bricolage Grotesque', ui-sans-serif, system-ui, sans-serif;
+  --font-sans: 'Source Sans 3 Variable', 'Source Sans 3', ui-sans-serif,
+    system-ui, sans-serif;
+  --font-mono: 'JetBrains Mono', ui-monospace, SFMono-Regular, Menlo, Monaco,
```

Stylelint (17.12.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)